first-year-parse/first-year.csv

```diff
@@ -1,36 +1,40 @@
-MA,MA10002,Maths II,--,3-1-0,4,V4,F142
-MA,MA10002,Maths II,--,3-1-0,4,C4,F116
-MA,MA10002,Maths II,--,3-1-0,4,C4,F142
-CY,CY11001,Chemistry,--,3-1-0,4,D4,F116
-CY,CY11001,Chemistry,--,3-1-0,4,D4,F142
-CY,CY11001,Chemistry,--,3-1-0,4,G3,F116
-CY,CY11001,Chemistry,--,3-1-0,4,G3,F142
-EE,EE11001,Electrical Technology,--,3-1-0,4,U4,F116
-EE,EE11001,Electrical Technology,--,3-1-0,4,U4,F116
-EE,EE11001,Electrical Technology,--,3-1-0,4,E4,F116
-EE,EE11001,Electrical Technology,--,3-1-0,4,E4,F116
-HS,HS13001,English,--,3-0-2,4,B3,F116
-HS,HS13001,English,--,3-0-2,4,B3,F142
-HS,HS13001,English,--,3-0-2,4,F4,F116
-HS,HS13001,English,--,3-0-2,4,F4,F142
-PH,PH11001,Physics,--,3-1-0,4,D4,NR122
-PH,PH11001,Physics,--,3-1-0,4,D4,NR222
-PH,PH11001,Physics,--,3-1-0,4,V4,NR122
-PH,PH11001,Physics,--,3-1-0,4,V4,NR222
-CS,CS10001,Programming and Data Structures,--,3-0-0,3,C4,NR122
-CS,CS10001,Programming and Data Structures,--,3-0-0,3,C4,NR222
-CS,CS10001,Programming and Data Structures,--,3-0-0,3,F4,NR122
-CS,CS10001,Programming and Data Structures,--,3-0-0,3,F4,NR222
-ME,ME10001,Mechanics,--,3-1-0,4,U4,NR123
-ME,ME10001,Mechanics,--,3-1-0,4,U4,NR124
-ME,ME10001,Mechanics,--,3-1-0,4,U4,NR223
-ME,ME10001,Mechanics,--,3-1-0,4,U4,NR224
-ME,ME10001,Mechanics,--,3-1-0,4,E4,NR123
-ME,ME10001,Mechanics,--,3-1-0,4,E4,NR124
-ME,ME10001,Mechanics,--,3-1-0,4,E4,NR223
-ME,ME10001,Mechanics,--,3-1-0,4,E4,NR224
+CS,CS10001,Programming and Data Structures,--,3-0-0,3,A3,NR122
+CS,CS10001,Programming and Data Structures,--,3-0-0,3,A3,NR222
+CS,CS10001,Programming and Data Structures,--,3-0-0,3,A3,NR121
+CS,CS10001,Programming and Data Structures,--,3-0-0,3,F4,NR122
+CS,CS10001,Programming and Data Structures,--,3-0-0,3,F4,NR222
+CY,CY11001,Chemistry,--,3-1-0,4,D3,F116
+CY,CY11001,Chemistry,--,3-1-0,4,D3,F142
+CY,CY11001,Chemistry,--,3-1-0,4,C4,F116
+CY,CY11001,Chemistry,--,3-1-0,4,C4,F142
+CY,CY11001,Chemistry,--,3-1-0,4,C4,V1
+EE,EE11001,Electrical Technology,--,3-1-0,4,U4,F116
+EE,EE11001,Electrical Technology,--,3-1-0,4,U4,F142
+EE,EE11001,Electrical Technology,--,3-1-0,4,E4,F116
+EE,EE11001,Electrical Technology,--,3-1-0,4,E4,F142
+EE,EE11001,Electrical Technology,--,3-1-0,4,E4,V1
+HS,HS13001,English,--,3-0-2,4,V3,F116
+HS,HS13001,English,--,3-0-2,4,V3,F142
+HS,HS13001,English,--,3-0-2,4,F3,F116
+HS,HS13001,English,--,3-0-2,4,F3,F142
+HS,HS13001,English,--,3-0-2,4,F3,V1
+MA,MA10002,Maths II,--,3-1-0,4,B3,F116
+MA,MA10002,Maths II,--,3-1-0,4,B3,F142
+MA,MA10002,Maths II,--,3-1-0,4,G3,F116
+MA,MA10002,Maths II,--,3-1-0,4,A3,F142
+MA,MA10002,Maths II,--,3-1-0,4,U4,V1
+MA,MA10002,Maths II,--,3-1-0,4,B3,NR122
+MA,MA10002,Maths II,--,3-1-0,4,B3,NR222
+MA,MA10002,Maths II,--,3-1-0,4,B3,NR121
+MA,MA10002,Maths II,--,3-1-0,4,G3,NR122
+MA,MA10002,Maths II,--,3-1-0,4,G3,NR222
+ME,ME10001,Mechanics,--,3-1-0,4,U3,NR122
+ME,ME10001,Mechanics,--,3-1-0,4,U3,NR222
+ME,ME10001,Mechanics,--,3-1-0,4,U3,NR121
+ME,ME10001,Mechanics,--,3-1-0,4,E3,NR122
+ME,ME10001,Mechanics,--,3-1-0,4,E3,NR222

```

Comment:
Can you please add this year's ME students time table for even section?

Reply:
A small change to be made. This year odd and even sections have the same room unlike previous years, so I have added an if needed  flag to the README